runtime/autoload/provider/pythonx.vim

```diff
@@ -53,13 +53,13 @@ function! s:check_interpreter(prog, major_ver, skip) abort
   endif
 
   " Try to load neovim module, and output Python version.
-  let prog_ver = system(a:prog . ' -c ' .
-        \ '''import sys; sys.stdout.write(str(sys.version_info[0]) + '.
-        \ '"." + str(sys.version_info[1])); '''.
+  let prog_ver = system([ a:prog , '-c' ,
+        \ 'import sys; sys.stdout.write(str(sys.version_info[0]) + '.
+        \ '"." + str(sys.version_info[1])); '.
         \ (a:major_ver == 2 ?
-        \   '''import pkgutil; exit(pkgutil.get_loader("neovim") is None)''':
-        \   '''import importlib; exit(importlib.find_loader("neovim") is None)''')
-        \ )
+        \   'import pkgutil; exit(pkgutil.get_loader("neovim") is None)':
+        \   'import importlib; exit(importlib.find_loader("neovim") is None)')
```

Comment:
operators (`:`, `?`) should be at the start of the next line. I will fix this when merging.